core/controllers/creator_dashboard.py

```diff
@@ -324,6 +324,11 @@ def _round_average_ratings(rating):
             self.values.update({
                 'topic_summary_dicts': topic_summary_dicts
             })
+        else:
+            self.values.update({
+                'topic_summary_dicts': None
```

Comment:
Technically, shouldn't this be an empty list?

Also I'm a bit confused. In creator-dashboard-backend-api-service.ts, if the variable is declared as TopicSummaryBackendDict[], shouldn't this error be detected?

Also, I'm surprised there isn't a backend test coverage error for these lines. Any idea why?

Reply:
> Technically, shouldn't this be an empty list?

done

> Also I'm a bit confused. In creator-dashboard-backend-api-service.ts, if the variable is declared as TopicSummaryBackendDict[], shouldn't this error be detected?

How could this error be detected, I mean typescript doesn't know what's there in the backend?

Also, we can assign `null` to any type.

> Also, I'm surprised there isn't a backend test coverage error for these lines. Any idea why?

There was a test for checking the response when the new structures are not enabled.